src/Rec/Rec.spec.ts

```diff
@@ -863,4 +863,31 @@ describe("Rec", () => {
       `)
         })
     })
+
+    describe("mergeInto", () => {
+        it("copies all entries when merged into empty Rec", () => {
+            const rec = { a: 1, b: 2, c: 3 }
+            expect(pipe(rec, Rec.mergeInto({}))).toStrictEqual(rec)
+        })
+
+        it("retains all entries when empty rec merged into Rec", () => {
+            const rec = { a: 1, b: 2, c: 3 }
+            expect(pipe({}, Rec.mergeInto(rec))).toStrictEqual(rec)
+        })
+
+        it("applies merge to original rec", () => {
```

Comment:
⛏️ I would call this test `it("overwrites entries with the same key in the 2nd rec")`